plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx

```diff
@@ -57,7 +67,7 @@ export const PaymentExtensionSuggestionListItem = ( {
 						{ ! hasIncentive && isWooPayments( extension.id ) && (
 							<StatusBadge status="recommended" />
 						) }
-						{ hasIncentive && extension._incentive && (
+						{ hasIncentive( extension ) && extension._incentive && (
```

Comment:
Is `extension._incentive` not redundant here?

Reply:
It is extremely redundant, but I get a TS warning about type safety if I don't include this check. 😢 I suppose I could add a linter ignore comment instead. WDYT?

Reply:
Managed to find a workaround for it!

Reply:
Glad you found a workaround :) Otherwise, putting a linter ignore comment sounded good too.